server/src/main/java/com/exacaster/lighter/application/sessions/SessionService.java

```diff
@@ -21,22 +23,31 @@
     private final ApplicationStorage applicationStorage;
     private final Backend backend;
     private final StatementHandler statementHandler;
+    private final SessionConfiguration sessionConfiguration;
 
-    public SessionService(ApplicationStorage applicationStorage, Backend backend, StatementHandler statementHandler) {
+    public SessionService(ApplicationStorage applicationStorage, Backend backend, StatementHandler statementHandler,
+            AppConfiguration appConfiguration) {
         this.applicationStorage = applicationStorage;
         this.backend = backend;
         this.statementHandler = statementHandler;
+        this.sessionConfiguration = appConfiguration.getSessionConfiguration();
     }
 
     public List<Application> fetch(Integer from, Integer size) {
         return applicationStorage.findApplications(ApplicationType.SESSION, from, size);
```

Comment:
We need to return Permanent sessions here too. So that we could track them on Lighter UI.